docs/configuration/overview.mdx

```diff
@@ -38,6 +38,7 @@ Payload is a *config-based*, code-first CMS and application framework. The Paylo
 | `rateLimit`          | Control IP-based rate limiting for all Payload resources. Used to prevent DDoS attacks and [more](/docs/production/preventing-abuse#rate-limiting-requests). |
 | `hooks`              | Tap into Payload-wide hooks. [More](/docs/hooks/overview) |
 | `plugins`            | An array of Payload plugins. [More](/docs/plugins/overview) |
+| `loggerOptions`     | Pino that will be passed through and used on Payload's logger. See [Pino Docs](https://getpino.io/#/docs/api?id=options) for more info on what is available.
```

Comment:
This reads funny to me, I would word it:
"Object used to configure log output settings using the Pino logging package. See..."